3d/commands.py

```diff
@@ -0,0 +1,24 @@
+#   Copyright 2016 Peter Kropf
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+
+import platform
+
+
+if platform.system() == 'Darwin':
+    openscad = '/Applications/OpenSCAD.app/Contents/MacOS/OpenSCAD'
+    inkscape = '/Applications/Inkscape.app//Contents/Resources/bin/inkscape'
```

Comment:
The double-slash is not needed, extra character left over from the cut and
paste.


On Fri, Aug 19, 2016 at 11:09 AM, Scott Bezek <notifications@github.com>
wrote:

> In 3d/commands.py
> <https://github.com/scottbez1/splitflap/pull/13#discussion_r75526919>:
>
> > +#   You may obtain a copy of the License at
> > +#
> > +#       http://www.apache.org/licenses/LICENSE-2.0
> > +#
> > +#   Unless required by applicable law or agreed to in writing, software
> > +#   distributed under the License is distributed on an "AS IS" BASIS,
> > +#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
> > +#   See the License for the specific language governing permissions and
> > +#   limitations under the License.
> > +
> > +import platform
> > +
> > +
> > +if platform.system() == 'Darwin':
> > +    openscad = '/Applications/OpenSCAD.app/Contents/MacOS/OpenSCAD'
> > +    inkscape = '/Applications/Inkscape.app//Contents/Resources/bin/inkscape'
>
> Is the double-slash needed here?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/scottbez1/splitflap/pull/13/files/27a01659e6c0123c91213a408cf76f70c955576b#r75526919>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAGEE4kzpVi43vN_6qoHeoaN3dn-ouyHks5qhfFSgaJpZM4JoFys>
> .
>